libsolidity/interface/StandardCompiler.cpp

```diff
@@ -769,6 +769,15 @@ Json::Value StandardCompiler::compileSolidity(StandardCompiler::InputsAndSetting
 			"Exception during compilation: " + boost::diagnostic_information(_exception)
 		));
 	}
+	catch (std::exception const& _e)
+	{
+		errors.append(formatError(
+			false,
+			"Exception",
+			"general",
+			"Unknown exception during compilation: " + (_e.what() ? string(_e.what()) : string{})
```

Comment:
Could use the same trick as below so as to avoid something like `Unknown exception during compilation: ` when `_e.what()` is null.

Reply:
even better: `"Unknown exception during compilation" + (_e.what() ? ": " + string(_e.what()) : string{"."}`

assuming `_e.what()` terminates with a full stop.